data/UrlStore.js

```diff
@@ -5,19 +5,21 @@
  * Copyright © 2020 Extremely Heavy Industries Inc.
  */
 
-import {LoadSupport, XH} from '@xh/hoist/core';
+import {XH, managed, LoadSupport} from '@xh/hoist/core';
 
 import {Store} from './Store';
 
 /**
  * A store with built-in support for loading data from a URL.
  */
-@LoadSupport
 export class UrlStore extends Store {
 
     url;
     dataRoot;
 
+    @managed
+    loadSupport = new LoadSupport(this);
```

Comment:
Should we add
```
async loadAsync(loadSpec) { this.loadSupport.loadAsync(loadSpec) }
```
to get the public API in there?  I saw in `LogViewerModel` we were calling `store.loadSupport.loadAsync(loadSpec)`. Any reason not to do so that I'm missing?